<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawStatistics</class>
 <widget class="QWidget" name="RawStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raw Node Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,2,0,3,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="id">
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="tableSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="localTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="toolTip">
        <string>Name of node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
       <property name="toolTip">
        <string>Identity of node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
       <property name="toolTip">
        <string>Address of node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MC</string>
       </property>
       <property name="toolTip">
        <string>Is this node multicast enabled.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string># Recv</string>
       </property>
       <property name="toolTip">
        <string>Number of packets received from this node.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string># Retry</string>
       </property>
       <property name="toolTip">
        <string>Number of retransmitted packets to this node.</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="remoteTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="toolTip">
        <string>Name of node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
       <property name="toolTip">
        <string>Identity of node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
       <property name="toolTip">
        <string>Address of node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MC</string>
       </property>
       <property name="toolTip">
        <string>Is this node multicast enabled.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string># Recv</string>
       </property>
       <property name="toolTip">
        <string>Number of packets received from this node.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string># Retry</string>
       </property>
       <property name="toolTip">
        <string>Number of retransmitted packets to this node.</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
